spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb

```diff
@@ -173,6 +179,9 @@ def close_stream(stream, style = :inprocess)
       end
 
       it "should create a new PowerShell manager host if the output stream handle is closed" do
+        # currently skipped as it can trigger an internal Ruby thread clean-up race
+        # its unknown why this test fails, but not the identical test against @stderr
+        skip('This test can cause intermittent segfaults in Ruby with w32_reset_event invalid handle')
```

Comment:
https://github.com/ruby/ruby/blob/ruby_2_3/lib/open3.rb#L201 is often where the error propagates from..